libs/styles/iframe.css

```diff
@@ -28,6 +28,8 @@
 
 @media (max-width: 1200px) {
   .dialog-modal.commerce-frame .milo-iframe{
-    padding-bottom:100vh;
+    padding-bottom: 100vh;
```

Comment:
We can remove this if we are overriding this in next line

Reply:
This has been added as a fallback for those browsers that do not support dvh.

Reply:
@aishwaryamathuria I have tested dvh in all standard browsers and update the code with the required changes.